### PR TITLE
#2 Extend the README.md file

### LINKED ISSUE #2: Readme Setup
As this is a newly open sourced Project it hsould have a readme.md.

This projects readme should ideally follow a similar structure to our other open-source projects

- https://github.com/orca-services/php-datatrans-xml-api-client/blob/master/README.md
- https://github.com/orca-services/nova-api-php
- https://github.com/orca-services/cakephp-heartbeat

### PR DESCRIPTION
**Pull request by Richard-Strittmatter**
Resolves #2

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,23 @@
 # Feature Flags plugin for CakePHP
 
+## Description
+
+This plugin can be used to enable or disable specific features in your code. This is very useful when you deploy your code to a production environment but you want to make sure that some features are only available at a later date and time.
+
+## Requirements
+
+* PHP 7.0+
```
Comment (ravage84) [inline README.md:9]: We should mention the required CakePHP version, too. Which is?

File: README.md [hunk 2]
```diff
@@ -1,11 +1,23 @@
...
 ## Installation
 
 You can install this plugin into your CakePHP application using [composer](https://getcomposer.org).
 
 The recommended way to install composer packages is:
 
+## Change log
+
+Please see [CHANGELOG](CHANGELOG.md) for more information on what has changed recently.
+
```
Comment (ravage84) [inline README.md:27]: This is at the wrong place.

**Review by ravage84 — CHANGES_REQUESTED** [review]
It would be useful to have the bare bones of other sections that need to be filled.

- TODOs
- Licence
etc.

See the some of the examples in #2 or any good PHP package.

**Comment by Richard-Strittmatter** [comment]
I'll add the license section in my second branch

**Review by ravage84 — APPROVED** [review]

**Pull request merged**